src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -165,15 +201,19 @@ Test that importing different video formats into an external texture and samplin
             },
           ],
         });
-        passEncoder2.setPipeline(pipeline);
-        passEncoder2.setBindGroup(0, bindGroup);
-        passEncoder2.draw(6);
-        passEncoder2.endPass();
+        passEncoder.setPipeline(pipeline);
+        passEncoder.setBindGroup(0, bindGroup);
+        passEncoder.draw(6);
+        passEncoder.endPass();
+      });
 
-        t.expectGPUError('validation', () => t.device.queue.submit([commandEncoder2.finish()]));
-      };
+      // This should trigger a microtask that destroys the GPUExternalTexture, which will cause
+      // a validation error when we try to use the destroyed GPUExternalTexture.
+      await Promise.resolve().then(() => {
+        t.expectGPUError('validation', () => t.device.queue.submit([commandEncoder.finish()]));
```

Comment:
nit: This puts two operations (finish and submit) in the callback, so we don't know which it came from.

Anyway I don't think the micro/tasks are quite issuing correctly in this code, so I'm looking at it locally.